pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -59,6 +59,9 @@ type BundleList struct {
 // BundleSpec defines the desired state of a Bundle.
 type BundleSpec struct {
 	// Sources is a set of references to data whose data will sync to the target.
+	// +listType=atomic
+	// +kubebuilder:validation:MinItems=1
+	// +kubebuilder:validation:MaxItems=100
```

Comment:
MaxItems is added to pass the cost requirements when CEL validations are added to the API.